### DISCUSSION TITLE
Installare wireguard client su raspi con hassio

### DISCUSSION THREAD

**Discussion by Antonio610821** [indomus/forum#697] [Networking]
Salve a tutti, sono nuovo e spero che sia la sezione giusta. Ho necessità di installare un client VPN Wireguard su un raspberri PI 4B dove è installato HassOs. L'addon , mi sembra di capire, si configura come server invece ho la necessità di configurare o installare un cliente per superare il nat degli operatori SIM. Vanno bene anche altri sistemi in quanto il server a cui si deve connettere è a mia gestione diretta.
Vi ringrazio e complimenti al forum
Antonio

**Comment by snakuzzo** [comment]
Hai provato a cercare su google? Perché cercando "hassos wireguard client", questo è il primo risultato...

https://github.com/bigmoby/addon-wireguard-client

**Reply by Antonio610821** [reply]
Si l'ho fatto  :(  solo che l'ho fatto con un ordine diverso dei termini e cioè wireguard client su hassio ....
comunque grazie. più tardi lo installo e faccio sapere.
Antonio

**Comment by Antonio610821** [comment]
Ciao, ho provato ad aggiungere il repository di wireguard client su hacs ma mi dice che :
Repository personalizzati

Repostitory structure for v0.1.7 is not compliant

Questo è il repository indicato:
https://github.com/bigmoby/addon-wireguard-client.git
 
Sicuramente sbaglio qualcosa ma non riesco a capire.
Grazie per il tuo cortese aiuto

**Reply by snakuzzo** [reply]
Devi aprire una issue sul progetto del client wireguard

**Reply by Antonio610821** [reply]
Ok grazie
Antonio
